tmva/pymva/src/RModelParser_Keras.cxx

```diff
@@ -71,6 +72,7 @@ const KerasMethodMap mapKerasLayer = {
    {"BatchNormalization", &MakeKerasBatchNorm},
    {"Reshape", &MakeKerasReshape},
    {"Concatenate", &MakeKerasConcat},
+   {"Swish", &MakeKerasSwish},
```

Comment:
nit: I believe the activation name internally is `swish`, and this will be case-sensitive.

Reply:
Thanks for pointing out. I will make the changes.